### ISSUE TITLE
Monitor errors

### ISSUE THREAD

**Issue by ccaceres** [Batlez/CloakBox#14]
Hi @Batlez 

Thanks for sharing, but when I try to run the safe browser, I get the following error:
![image](https://github.com/user-attachments/assets/b47d8448-9884-484d-8113-1e297e8515e9)

Any idea How can I fix it? Also, how can I make the webcam undetectable?

Thanks a lot

**Comment by Batlez** [comment]
@ccaceres I had this issue on mine aswell, but I followed this guide and it fixed the issue for me. Add me on Discord: Croakq



    open this link -> https://www.entechtaiwan.com/util/moninfo.shtm

    download the software

    install it and open it

    go on top left click on file option and click on create Imf file and save it

    once done remember the location you saved that file on.

    now go to system settings,then go to update and security and go to recovery and click restart now

    when you are in recovery click on troubleshoot and select advanced options and select startup settings

    on the new window opened (if it ask you to select account then select your account) after that press 7 on your keyboard and your
    computer may restart

    now go to device manager, then drop down the monitor tab, then right click on the "generic non-pnp monitor" and click update driver

    now select "browse my computer drivers" and then click "let me pick" then click on have disk.

    now click on "browse" and then select the imf file we saved in starting and once it is installed let the device manager refresh itself.

    now again right click on what so ever name is there in place of "generic non-pnp monitor" and go to update driver but this time click on select driver automatically

    once it is done just run msb again and check the error must get solved ❤️✌️

**Comment by Batlez** [comment]
And It's also working on Safe Exam Browser V3.8.0
![{EA076D9E-47A2-402C-A0B8-999CC2DAAEF9}](https://github.com/user-attachments/assets/eb193fe8-ed2b-40e8-b494-9519436e94d3)

**Comment by Ahmeddmemon** [comment]
Works :)

**Comment by Ahmeddmemon** [comment]
Works, thank you :) 

Also one question, is there a way to make the cursor stop/pause on the screen of VM? like we can do it in VMware. Because won't it be suspicious dragging the cursor to the side everytime we want to search something on the MAIN PC

**Comment by Batlez** [comment]
Yes you can enable mouse integration. 
![{2C41E81E-297D-4A99-8872-0F4CEA2C0747}](https://github.com/user-attachments/assets/ae716d23-8bda-4ee7-8780-e805d9169f39)

**Comment by SimpSlayerr** [comment]
Yes thank you, I figured it out. Its similar like VMware. Right CTRL locks/unlocks the cursor inside the Vbox

**Comment by Batlez** [comment]
Please try out the new version that came out and let me know if you still have issues!
